content/en/blog/2024/elastic-contributes-continuous-profiling-agent.md

```diff
@@ -0,0 +1,150 @@
+author:
+  >- # If you have only one author, then add the single name on this line in quotes.
+  [Bahubali Shetti](https://github.com/bshetti) (Elastic), [Alexander
+  Wert](https://github.com/AlexanderWert) (Elastic), [Morgan
+  McLean](https://github.com/mtwo) (Splunk), [Ryan
+  Perry](https://github.com/Rperry2174) (Grafana)
+draft: true # TODO: remove this line once your post is ready to be published
+# canonical_url: http://somewhere.else/ # TODO: if this blog post has been posted somewhere else already, uncomment & provide the canonical URL here.
+issue: https://github.com/open-telemetry/community/issues/1918
+sig: Profiling SIG
+---
+
+Following significant collaboration between
+[Elastic](https://www.elastic.co/observability-labs/blog/elastic-donation-proposal-to-contribute-profiling-agent-to-opentelemetry)
+and [OpenTelemetry's profiling community](/blog/2024/profiling/), which included a
+thorough review process, we’re excited to announce that the OpenTelemetry
+project has accepted
+[Elastic's donation of its continuous profiling agent](https://github.com/open-telemetry/community/issues/1918).
+
+This marks a significant milestone in establishing profiling as a core telemetry
+signal in OpenTelemetry. Elastic’s [eBPF based](https://ebpf.io/) profiling agent
+observes code across different programming languages and runtimes, third-party
+libraries, kernel operations, and system resources with low CPU and memory
+overhead in production. Both, SREs and developers can now benefit from these
+capabilities: quickly identifying performance bottlenecks, maximizing resource
+utilization, reducing carbon footprint, and optimizing cloud spend.
+
+Elastic’s decision to contribute the project to OpenTelemetry was made to
+accelerate OpenTelemetry’s mission and enable effective observability through
+high-quality, portable telemetry. This collaboration also shows the commitment
+to vendor neutrality and community-driven development enhancing the overall
+profiling and observability ecosystems.
+
+The Elastic team would like to thank the OpenTelemetry community for the great
+and constructive cooperation on the donation proposal. We look forward to
+jointly establishing continuous profiling as an integral part of OpenTelemetry.
+
+With today’s acceptance, Elastic’s continuous profiling agent will be
+contributed to OpenTelemetry. This agent will now be jointly supported by both
+Elastic’s team as well as a diverse set of official maintainers from different
+companies:
+
+- Dmitry Filimonov (Grafana Labs)
+- Felix Geisendörfer (Datadog)
+- Jonathan Halliday (Red Hat)
+- Christos Kalkanis (Elastic)
+
+## What is continuous profiling?
+
+[Continuous profiling](https://www.cncf.io/blog/2022/05/31/what-is-continuous-profiling/)
+is a technique used to understand the behavior of a software application by
+collecting information about its execution. This includes tracking the duration
```

Comment:
This paragraph is also true for just `profiling` instead of `continuous profiling`.
```suggestion
collecting information about its execution over time. This includes tracking the duration
```
Alternative suggestion: `over time` -> `as a function of time`.
(Just ideas, not sure how a native speaker would phrase this, tbh.)